HostsFileUpdaterScript

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Hosts file updater
+# You will probably need root or su rights to access /etc/hosts
+# modified by Valkiry to be used withb Steven Black's Unified Hosts files.
+#
+# Debian users will need the package "sysutils" as this script uses dos2unix
+# $ apt-get install sysutils
+#
+HFSERVER="https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts"
+HFILE="hosts.txt"
+ORIGFILE="/etc/hosts.original"
+
+clear
+echo "-------------------------------------------------------------"
+echo "This script will update your Hosts file to the latest version of StevenBlack's Unified hosts = (adware + malware)"
+echo "Your original Hosts file will be renamed to $ORIGFILE"
+echo "-------------------------------------------------------------"
+echo ""
+
+if [ ! -f "$ORIGFILE" ] ; then
+  echo "Backing up your previous hosts file.." 
+  cp -v /etc/hosts $ORIGFILE # I like verbose file operations.  Can be less verbose if necessary.
+fi
+
+echo "Retrieving $HFILE from $HFSERVER"
+echo ""
+wget -O /tmp/$HFILE $HFSERVER/$HFILE
```

Comment:
Okay so this pull request is incomplete then ... 
Can't wait to see the next update :wink: